app/views/projects/index.html.erb

```diff
@@ -38,4 +38,23 @@
   </div>
 </div>
 
+<template id="projectPlaceholder">
+  <div class="project-placeholder card">
+    <% 4.times do |i| %>
+      <div class="placeholder-element line-<%= i %>"></div>
+    <% end %>
+    <% 3.times do |i| %>
+      <div class="placeholder-element circle circle-<%= i %>"></div>
+    <% end %>
+  </div>
+</template>
+
+<template id="projectEndOfList">
+  <div class="project-list-end-placeholder">
+    <i class="fas fa-flag-checkered"></i>
+    <span><%= t('.end_of_list_placeholder') %></span>
```

Comment:
is it correct?

Reply:
Yes, it will append the file name and folder name. All others are in same format. 
<img width="759" alt="Screenshot 2022-04-13 at 12 07 16" src="https://user-images.githubusercontent.com/47317017/163155492-5ccf694a-48ae-41ad-9a34-07b378d11a98.png">